contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraPOJOOutputOperator.java

```diff
@@ -162,6 +163,30 @@ public void activate(Context.OperatorContext context)
       }
       getters.add(getter);
     }
+    super.activate(context);
+  }
+
+  private void populateFieldInfosFromPojo(ColumnDefinitions rsMetaData)
+  {
+    fieldInfos = Lists.newArrayList();
+    Field[] fields = pojoClass.getDeclaredFields();
+    for (int i = 0; i < rsMetaData.size(); i++) {
+      String columnName = rsMetaData.getName(i);
+      String pojoField = getMatchingField(fields, columnName);
+      if (pojoField != null && pojoField.length() != 0) {
```

Comment:
what happens if pojoField is null? Please check if this will cause problems when no matching field is found.